library/user/reconnaissance/blue_clues/payload.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+# Title: Blue Clues
+# Author: Brandon Starkweather
+
+
+# --- 1. SETUP ---
+LOOT_DIR="/root/loot/blue_clues"
+
+# Hardware Control Functions
+set_global_color() {
+    # $1=R, $2=G, $3=B
+    for dir in up down left right; do
+        if [ -f "/sys/class/leds/${dir}-led-red/brightness" ]; then
+            echo "$1" > "/sys/class/leds/${dir}-led-red/brightness"
+            echo "$2" > "/sys/class/leds/${dir}-led-green/brightness"
+            echo "$3" > "/sys/class/leds/${dir}-led-blue/brightness"
+        fi
+    done
+}
+
+set_led() {
+    STATE="$1"
+    if [ "$STATE" -eq 1 ]; then
+        if [ "$FB_MODE" -eq 2 ] || [ "$FB_MODE" -eq 4 ]; then
+            set_global_color 255 0 0 # Red
+        fi
+    elif [ "$STATE" -eq 2 ]; then
+        set_global_color 0 255 0 # Green
+    else
+        set_global_color 0 0 0 # Off
+    fi
+}
+
+do_vibe() {
```

Comment:
This should really be replaced with the VIBRATE command. For example:

VIBRATE 'Vibe1Sec:d=4,o=6,b=60:c'